Makefile

```diff
@@ -16,14 +16,18 @@ clean:
 test: test-local
 	set -eu; \
 	for p in $(PACKAGES); do \
-		(cd $$p; go test $(RUN_VIA_SUDO) -v .); \
+		if $p = user && go version | grep -qv go1.17; then \
```

Comment:
No need to support Go <= 1.20

Reply:
The other modules still have go1.17 as minimum version; I didn't want to change that as part of this PR, but we can revisit that in a follow up